src/main/java/frc/robot/maps/GastonMap.java

```diff
@@ -90,20 +90,25 @@ public IntakeMap getIntakeMap() {
 
     @Override
     public ClimberMap getLeftClimberMap() {
-        final DigitalInput leftUpperLimit = new DigitalInput(1);
-        final DigitalInput leftLowerLimit = new DigitalInput(2);
-        final PIDSparkMax leftMotor = new PIDSparkMax(9, MotorType.kBrushless);
-        leftMotor.validateCurrent(CLIMBER_CURRENT_LIMIT);
-        return new ClimberMap(leftMotor, leftUpperLimit::get, leftLowerLimit::get);
+        final PIDSparkMax extendMotor = new PIDSparkMax(9, MotorType.kBrushless);
+
+        final PIDSparkMax rotateMotor = new PIDSparkMax(18, MotorType.kBrushless);
+
+        extendMotor.validateCurrent(CURRENT_LIMIT);
```

Comment:
This should also set the current limit on the motor controller to help ensure we don't break anything.